ext/intl/grapheme/grapheme_string.c

```diff
@@ -84,6 +84,7 @@ PHP_FUNCTION(grapheme_strpos)
 	char *haystack, *needle;
 	size_t haystack_len, needle_len;
 	const char *found;
+	char *locale = "";
```

Comment:
You should initialise these with `= NULL;`, the empty string is done in a code segment like this, and will not be happy when you free it.

Reply:
Thank you very much. Indeed. `locale` should `= NULL;`. Therefore, signature is wrong.